apps/prairielearn/src/schemas/schemas/infoQuestion.json

```
@@ -227,10 +227,10 @@
           "type": "boolean"
         },
         "gradedFiles": {
-          "description": "The list of files or directories that will be copied for grading.",
+          "description": "The list of files or directories that will be copied to the submission.",
```

Nit, let's use the same language as the docs:

```suggestion
          "description": "The list of files or directories that will be copied out of the workspace container when saving a submission.",
```